components/contracts/missionsInLocation.ts

```diff
@@ -16,310 +16,634 @@
-            "ccbde3e2-67e7-4534-95ec-e9bd7ef65273",
-            "1f785def-03b7-4340-af7e-2f5831e77eb5",
-            "45c831c4-b455-4d21-90f3-6f09b28ee01b",
-        ],
-        LOCATION_BANGKOK_ZIKA: [],
-        LOCATION_COLORADO: [
-            "e6f4d3a4-9a33-4bd9-b761-da297069cf8c",
-            "c5d88e8c-437b-476b-afe2-d94aa4293502",
-            "4186dd23-1cfc-4ba0-9863-9f19f7cba249",
-        ],
-        LOCATION_COLORADO_RABIES: [],
-        LOCATION_HOKKAIDO: [
-            "115425b1-e797-47bf-b517-410dc7507397",
-            "e96fb040-a13f-466c-9d96-c8f3b2b8a09a",
-            "a1e5f4f4-ea9c-4a42-b826-50a212026d50",
-            "85a2b618-2e3c-444f-931c-b89d566e45f7",
-            "88451dd9-4b57-441e-9eab-e20b9879bafa",
-        ],
-        LOCATION_HOKKAIDO_MAMUSHI: [],
-        LOCATION_HOKKAIDO_FLU: [],
-        LOCATION_NEWZEALAND: [
-            "e1e86206-d3f0-a819-e477-3d80e55e8a40",
-            "3efc73f9-33f0-4af6-9508-7208e6851394",
-        ],
-        LOCATION_MIAMI: [
-            "719ee044-4b05-4bd9-b2bb-75029f6d2a35",
-            "5284cb9f-9bdd-4b00-99c3-0b5939b01818",
-            "d0f44e71-6eab-4af4-9484-78d61dbe376a",
-            "fca539ff-1b1a-4c04-93e0-03b9b902f86c",
-            "782a2849-14a2-4cd4-99fc-ddacaeaba2dd",
-            "be3ea01f-ec56-4fcb-95ec-164a1d9980f3",
-            "066ce378-0418-452a-b02e-a5e4ee711096",
-        ],
-        LOCATION_COLOMBIA: [
-            "390ba7b6-de27-464a-b8af-6d0ff54c2aec",
-            "70150cd2-ef76-4ba8-80cc-b1e63871b030",
-            "be567ad3-23f4-4d0b-9d2e-b261ea845ef0",
-            "a5e81878-0eae-4bcf-af9b-9a7e7833f85d",
-            "757fd132-0300-45ec-b5bd-bdd48c543b5c",
-            "41ecf8ce-dfd4-4c08-8f44-52dedc3f089a",
-            "d336d894-024a-4cd4-9867-dee7de70ee79",
-        ],
-        LOCATION_MUMBAI: [
-            "667f48a3-7f6b-486e-8f6b-2f782a5c4857",
-            "9badee3e-0014-46b1-9ef6-edf8858ba038",
-            "b6a6330a-301a-4e8e-a26f-0f3e0ea809b5",
-            "4a62b328-dfe7-4956-ac0f-a3a8990fce26",
-            "a10472e4-0eb3-451d-814d-38837dd0f407",
-            "ae0bd6cd-7062-4336-8cb0-5fafad3d0f4f",
-            "b47f34cb-6537-421c-8fc8-720a4a118540",
-        ],
-        LOCATION_NORTHAMERICA: [
-            "218302a3-f682-46f9-9ffd-bb3e82487b7c",
-            "d1b9250b-33f6-4712-831b-f33fa11ee4d8",
-            "1d5dcf3e-9682-4e32-ac11-ad6586daa456",
-            "74e6b561-ff1a-4742-9a7b-890b7818c796",
-            "15b7ad4e-565a-4fdb-b669-c9a68176e665",
-            "fe088a10-5dbf-460f-bbe2-6b55e7a66253",
-        ],
-        LOCATION_NORTHSEA: [
-            "e63eeb62-29ef-428d-b003-ea043b1f11f9",
-            "b66f151d-47a7-4681-a403-c48a46916224",
-            "dbb0e22d-084b-4b57-8616-42290982fd90",
-            "4fbfae2e-a5e7-4b79-b008-94f6cbcb13cb",
-            "3721e543-b5e6-4af8-a4fc-c92e9a4453bd",
-            "8c6daf5e-5974-4438-af20-71ff570c7ff3",
+export const missionsInLocation = {
```

Comment:
Explicitly type this object

Reply:
Does [this](https://github.com/thepeacockproject/Peacock/pull/550/files#diff-b388de6427dba20db03a608266dd354bb8fb9769041dc9c43d5810e63d75920aR649) not _satisfy_ that?